tools/cli/workflowCommands.go

```diff
@@ -1792,12 +1798,59 @@ func getResetEventIDByType(ctx context.Context, c *cli.Context, resetType, domai
 		if err != nil {
 			return
 		}
+	case "DecisionStartedTime":
+		earliestTime := parseTime(c.String(FlagEarliestTime), 0)
+		resetBaseRunID, decisionFinishID, err = getEarliestDecisionID(ctx, domain, wid, rid, earliestTime, frontendClient)
+		if err != nil {
+			return
+		}
 	default:
 		panic("not supported resetType")
 	}
 	return
 }
 
+func getEarliestDecisionID(ctx context.Context, domain string, wid string, rid string, earliestTime int64, frontendClient workflowserviceclient.Interface) (resetBaseRunID string, decisionFinishID int64, err error) {
+	resetBaseRunID = rid
```

Comment:
nit: seems like `resetBaseRunID` is just `rid` in this case, why we still need to return it?

Reply:
Good catch, will remove it.